File: replace_smtp.js
const fs = require('fs');

const smtpDetails = \### SMTP (Resend)
O SMTP externo via Resend JÁ ESTÁ CONFIGURADO no Supabase Dashboard:
- **Host:** smtp.resend.com
- **Port:** 587
- **Username:** resend
- **Status:** ✅ CONFIGURADO E FUNCIONANDO
- **Domínio de envio:** mxos.com.br
- **Limite:** sem limite de emails/hora (substituiu o limite de 3-4/hora do Supabase gratuito)\;

// CLAUDE.md
let claude = fs.readFileSync('CLAUDE.md', 'utf8');
claude = claude.replace(/### Aviso SMTP[\\s\\S]*?\\*\\*Supabase Dashboard → Authentication → Settings → SMTP Settings\\*\\*/, smtpDetails);
claude = claude.replace(/- SMTP externo \\(Resend ou AWS SES\\) antes de produção\\./, '- [x] SMTP externo via Resend já configurado.');
claude = claude.replace(/- SMTP externo \\(Resend ou AWS SES\\) antes de produção/, '- [x] SMTP externo via Resend já configurado');
claude = claude.replace(/- SMTP externo \\(Resend ou AWS SES\\)/, '- [x] SMTP externo via Resend configurado');
fs.writeFileSync('CLAUDE.md', claude);

// GEMINI.md
let gemini = fs.readFileSync('GEMINI.md', 'utf8');
gemini = gemini.replace(/### Aviso SMTP[\\s\\S]*?Dashboard → Authentication → Settings → SMTP Settings/, smtpDetails);
gemini = gemini.replace(/Lighthouse audit · SMTP externo · CI\\/CD/, 'Lighthouse audit · CI/CD');
fs.writeFileSync('GEMINI.md', gemini);

// ANTIGRAVITY.md
let antigravity = fs.readFileSync('ANTIGRAVITY.md', 'utf8');
antigravity = antigravity.replace(/\\| 6 \\| \\*\\*SMTP externo\\*\\* \\| Supabase gratuito limita 3-4 emails\\/hora[\\s\\S]*?\\| Antes de lançar \\|/, '| 6 | **SMTP externo** | ✅ **CONFIGURADO** via Resend (smtp.resend.com). Domínio: mxos.com.br. Sem limite de envios. | Concluído |');
antigravity = antigravity.replace(/Lighthouse audit · SMTP externo · CI\\/CD/, 'Lighthouse audit · CI/CD');
fs.writeFileSync('ANTIGRAVITY.md', antigravity);

// ARBO_FASE2.md
let arbo = fs.readFileSync('ARBO_FASE2.md', 'utf8');
arbo = arbo.replace(/\\| 🟢 Baixa \\| \\*\\*SMTP externo\\*\\* \\| Resend ou AWS SES para não travar com limite de 3-4 emails\\/hora do Supabase gratuito \\|/, '| ✅ Concluído | **SMTP externo** | Resend configurado (smtp.resend.com) no domínio mxos.com.br sem limite de emails/hora |');
arbo = arbo.replace(/- SMTP externo \\(Resend ou AWS SES\\) antes de produção\\./, '- [x] SMTP externo via Resend já configurado.');
fs.writeFileSync('ARBO_FASE2.md', arbo);

// CLAUDE_HISTORICO.md
let historico = fs.readFileSync('CLAUDE_HISTORICO.md', 'utf8');
historico = historico.replace(/- SMTP externo \\(Resend ou AWS SES\\)/, '- [x] SMTP externo via Resend configurado');
fs.writeFileSync('CLAUDE_HISTORICO.md', historico);

console.log('Done!');

File: replace_smtp.py
import re

smtp_details = '''### SMTP (Resend)
O SMTP externo via Resend JÁ ESTÁ CONFIGURADO no Supabase Dashboard:
- **Host:** smtp.resend.com
- **Port:** 587
- **Username:** resend
- **Status:** ✅ CONFIGURADO E FUNCIONANDO
- **Domínio de envio:** mxos.com.br
- **Limite:** sem limite de emails/hora (substituiu o limite de 3-4/hora do Supabase gratuito)'''

def replace_in_file(filepath, replacements):
    try:
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        for p, r in replacements:
            content = re.sub(p, r, content)
        with open(filepath, 'w', encoding='utf-8') as f:
            f.write(content)
    except Exception as e:
        print(f'Error in {filepath}: {e}')

replace_in_file('CLAUDE.md', [
    (r'### Aviso SMTP[\\s\\S]*?\\*\\*Supabase Dashboard → Authentication → Settings → SMTP Settings\\*\\*', smtp_details),
    (r'- SMTP externo \\(Resend ou AWS SES\\) antes de produção\\.', '- [x] SMTP externo via Resend já configurado.'),
    (r'- SMTP externo \\(Resend ou AWS SES\\) antes de produção', '- [x] SMTP externo via Resend já configurado'),
    (r'- SMTP externo \\(Resend ou AWS SES\\)', '- [x] SMTP externo via Resend configurado')
])

replace_in_file('GEMINI.md', [
    (r'### Aviso SMTP[\\s\\S]*?Dashboard → Authentication → Settings → SMTP Settings', smtp_details),
    (r'Lighthouse audit · SMTP externo · CI/CD', 'Lighthouse audit · CI/CD')
])

replace_in_file('ANTIGRAVITY.md', [
    (r'\\| 6 \\| \\*\\*SMTP externo\\*\\* \\| Supabase gratuito limita 3-4 emails/hora[\\s\\S]*?\\| Antes de lançar \\|', '| 6 | **SMTP externo** | ✅ **CONFIGURADO** via Resend (smtp.resend.com). Domínio: mxos.com.br. Sem limite de envios. | Concluído |'),
    (r'Lighthouse audit · SMTP externo · CI/CD', 'Lighthouse audit · CI/CD')
])

replace_in_file('ARBO_FASE2.md', [
    (r'\\| 🟢 Baixa \\| \\*\\*SMTP externo\\*\\* \\| Resend ou AWS SES para não travar com limite de 3-4 emails/hora do Supabase gratuito \\|', '| ✅ Concluído | **SMTP externo** | Resend configurado (smtp.resend.com) no domínio mxos.com.br sem limite de emails/hora |'),
    (r'- SMTP externo \\(Resend ou AWS SES\\) antes de produção\\.', '- [x] SMTP externo via Resend já configurado.')
])

replace_in_file('CLAUDE_HISTORICO.md', [
    (r'- SMTP externo \\(Resend ou AWS SES\\)', '- [x] SMTP externo via Resend configurado')
])

print('Done!')
